tests/formatter_tests.ts

```diff
@@ -354,10 +355,10 @@ function multiStaves(options: TestOptions): void {
   // Two helper functions to calculate the glyph's width.
   // Should these be static methods in Glyph or Font?
   function glyphPixels(): number {
-    return 96 * (38 / (Flow.DEFAULT_FONT_STACK[0].getResolution() * 72));
+    return 96 * (38 / (Flow.MUSIC_FONT_STACK[0].getResolution() * 72));
   }
   function glyphWidth(vexGlyph: string): number {
-    const glyph: FontGlyph = Flow.DEFAULT_FONT_STACK[0].getGlyphs()[vexGlyph];
+    const glyph: FontGlyph = Flow.MUSIC_FONT_STACK[0].getGlyphs()[vexGlyph];
```

Comment:
This would need updates to the Wiki too, I think.

Reply:
Yes, I will make sure to update the wiki.

**Are you okay with this change though?** 

My reasoning is that many classes have properties that include the word "font". But it is not clear whether the font is for music engraving purposes, or for text display purposes.

My initial idea was to have Element have two properties: `musicFont` and `textFont`.

```
  /** The font used to render musical glyphs (e.g., treble clef).*/
  protected musicFont!: Font;

  /** Some elements include text. You can customize the font family, size, weight, and style. */
  protected textFont?: Required<FontInfo>;
```

However, there is a descendant of Element that has its own .textFont property! :-( That class is ChordSymbol.

Perhaps that class should be renamed (TextFont => TextFontMetrics, or => TextFontRegistry)? @AaronDavidNewman 

Currently in this PR, the Element class has a .musicFont and a .font property, where .font is assumed to be for text purposes.

This is not bad, because CanvasContext and SVGContext also have a .font accessor which is for text rendering purposes, to adhere to the CanvasRenderingContext2D API. But perhaps it would be most clear to rename it to `textFont`, and then have ChordSymbol rename its member to `textFontMetrics`...


To further improve the naming, I went from DEFAULT_FONT_STACK to **DEFAULT_MUSIC_FONT_STACK**!

I then realized there are only three properties starting with `DEFAULT_`, even though there are all sorts of defaults all over VexFlow. So I figured we could drop the DEFAULT_ prefix for those three (all related to music fonts).

The breaking changes will be added to the CHANGELOG.